pkg/iplister/reader/file/file_test.go

```diff
@@ -0,0 +1,52 @@
+package file
+
+import (
+	"context"
+	"io"
+	"os"
+	"path"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+const (
+	testFile    = "iplists.yaml"
+	testContent = `
+iplists:
+  jira:
+    - "1.2.3.4/32"
+    - "5.6.7.8/28"
+  other:
+    - "101.102.103.104/32"
+    - "101.102.103.104/28"
+`
+)
+
+func TestNew(t *testing.T) {
+	t.Parallel()
+
+	f := New(testFile)
+	assert.Equal(t, testFile, f.filename)
+}
+
+func TestData(t *testing.T) {
```

Comment:
This is a weird test given that all the `Data()` function does is `return os.Open(f.filename)`.

I've made this test do the full thing so that if there was any changes to `Data()` it would still verify the `io.ReadCloser` it returns is good